setup.cfg

```diff
@@ -1,3 +1,40 @@
 [flake8]
 ignore = E128,E811,W503,E203
 max-line-length = 100
+
+[metadata]
+name = linkstatus
+author = Nikhil Dhandre
+author_email = nik.digitronik@live.com
+license = GPLv3
```

Comment:
If possible please include meta
```
license-file = LICENSE
```

Reply:
Based upon https://setuptools.readthedocs.io/en/latest/setuptools.html#metadata
`license_file = LICENSE` added. This meta accepts an only string.